video-course/src/app/core/header/header.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="container-fluid">
   <div class="row bg-secondary">
     <vc-logo class="col-sm text-left"></vc-logo>
-    <vc-login class="col-sm text-right"></vc-login>
-    <vc-logoff class="col-sm text-right"></vc-logoff>
+    <vc-login class="col-sm text-right" *ngIf='isDisplayLogin === "true"'></vc-login>
```

Comment:
isDisplayLogin is boolien ? so no need to check
isDisplayLogin === "true"'
just :
 *ngIf='isDisplayLogin'